src/igvfd/tests/test_types_analysis_set.py

```diff
@@ -179,5 +179,6 @@ def test_workflows(testapp, analysis_set_with_workflow, matrix_file_with_base_wo
             'file_set': analysis_set_with_workflow['@id']
         }
     )
-    res = testapp.get(analysis_set_with_workflow['@id']).json.get('workflows')
-    assert res == ['/workflows/IGVFWF0000WRKF/']
+    res = testapp.get(analysis_set_with_workflow['@id'])
+    assert set([workflow['@id'] for workflow in res.json.get('workflows')]
+               ) == {'/workflows/IGVFWF0000WRKF/'}
```

Comment:
I am not sure why changing it to a set(). The calculation step https://github.com/IGVF-DACC/igvfd/blob/210ca41e8b0efed227aaae99d3f59dd35404f1f9/src/igvfd/types/file_set.py#L486 starts with a set and returns a list(set()).